.github/workflows/containerd.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         # ║ hcshim           │           │ runhcs  ║
         # ╚══════════════════╧═══════════╧═════════╝
         os: [ubuntu-20.04, windows-2019]
-        version: [main, v1.6.12]
+        version: [main, release/1.6, release/1.7]
```

Comment:
Note that `release/1.6` and `release/1.7` are branches rather than tags, so we won't be testing _specific_ releases of containerd but rather development versions of the next version in the series.

I think this is likely okay and overall reduces toil of maintaining the workflow here.

Reply:
CI doesn't like slash in branch name...

Reply:
Might be worth trying the replace in the other direction: use something like "release_1.6" here and then update the `actions/checkout@v2` `ref` field on line 73 with `${{ replace(matrix.version, '_', '/') }}`?